### ISSUE TITLE
您好，请问您的text_recognition loss如何实现的？

### ISSUE THREAD

**Issue by Chenglu0426** [YuzheZhang-1999/DiffTSR#13]
您好：我们注意到您的实现中有个text_recognization loss，会根据每个字在2d空间中的embedding之间的距离进行加权，从而计算loss。但是原作品中使用的是英文字符，中文字符如何计算embedding之间的距离？

**Comment by YuzheZhang-1999** [comment]
感谢您的关注。
关于该问题，正如我在附加材料section 1.1中的第10行所述，text_recognization loss = cross_entropy(F(x_0), F(x_gt))，其中F是一个预训练好的中英文字符识别网络[1]，虽然该模型F的原始论文是用于英文字符识别的，但是经过修改其字符码本、微调模型结构并重新训练即可用于中英文字符识别。该论文的作者也提供了该模型在中英文字符上训练好的权重和模型[2]，你可以用作你的字符特征提取模型；此外，你同样可以参考本工作的做法，在自己的训练集上进行训练，从而适配不同语言、不同字符码本和字符长度~

[1]Jingye Chen, Bin Li, and Xiangyang Xue. Scene text telescope: Text-focused scene image super-resolution.
[2]https://github.com/FudanVI/benchmarking-chinese-text-recognition/tree/main/models
